java/test/jmri/configurexml/valid/SensorLegacyName.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://www.jmri.org/xml/XSLT/panelfile.xsl"?>
+
+<!-- sensor elements with a legacy name fail -->
```

Comment:
This seems to be a copy/paste error. I guess this file should succeed, not fail?